src/Tribe/Tickets.php

```diff
@@ -1054,6 +1054,11 @@ public function save_checkin_details( $attendee_id, $qr ) {
 				'author' => get_current_user_id(),
 			];
 
+			if ( ! empty( $qr ) ) {
+				// Save the latest date in which a ticket was scanned with the APP.
+				tribe_update_option( 'tec_tickets_plus_app_last_checkin_time', time() );
```

Comment:
Thanks for the heads up! For now, we want the server time to know when was the last time they used the app to promote stuff or ask questions about the experience with the app. It will not be altered if the user changes timezone on their site or similar.